src/dom/index.js

```diff
@@ -52,10 +52,21 @@ export function setAccessor(node, name, old, value, isSvg) {
 		}
 		if (value && typeof value==='object') {
 			if (typeof old!=='string') {
-				for (let i in old) if (!(i in value)) node.style[i] = '';
+				for (let i in old) if (!(i in value)) {
+					if (i.indexOf('--') === 0) {
```

Comment:
just re-read the original thread, i was wrong since setProperty() uses CSS case rather than camelCase. poop!